package.json

```diff
@@ -81,5 +81,6 @@
   },
   "dependencies": {
     "body-scroll-lock": "^3.1.5"
-  }
-}
\ No newline at end of file
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
What's this for? Does this enforce the package manager for package users?

Reply:
Yeah, it was automatically added by corepack. I'll remove it since it doesn't matter.